apps/test/unit/templates/AppViewTest.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../util/configuredChai';
+import {UnconnectedAppView as AppView} from '@cdo/apps/templates/AppView';
+
+describe('AppView', () => {
+  const DEFAULT_PROPS = {
+    hideSource: false,
+    isResponsive: false,
+    pinWorkspaceToBottom: false,
+    visualizationColumn: null,
+    onMount: () => {},
+  };
+
+  it('no class names', () => {
```

Comment:
This is very small, but I might go into a little more detail in your `it` statements (this test and below). I wasn't sure what each test was highlighting until I read through all of the `expect`s. However, the `expect`s themselves might do a better job than trying to boil down something descriptive enough for each `it` statement, so not sure that those need to be changed right now. 

Approved, and this looks great! 🎉 yay ES6